src/categorical_algebra/FinSets.jl

```diff
@@ -1464,4 +1465,31 @@ join(A::SubFinSet{Int}, B::SubFinSet{Int}, ::SubOpBoolean) =
 top(X::FinSet{Int}, ::SubOpBoolean) = SubFinSet(trues(length(X)))
 bottom(X::FinSet{Int}, ::SubOpBoolean) = SubFinSet(falses(length(X)))
 
+# SubVarSets
+
+""" Subset of a finite varset.
+"""
+const SubVarSet{T} = Subobject{VarSet{T}}
+
+Subobject(X::VarSet{T}, f::Union{AbstractVector, UnitRange}) where T = 
```

Comment:
Looks like you have two copies of the same method, which is why this one isn't covered. Also, `AbstractVector` will probably suffice instead of `Union{AbstractVector,UnitRange}`.